<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1388</width>
    <height>931</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dave Automatic Trading for Education v1.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>davetrading.png</normaloff>davetrading.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>39</y>
      <width>751</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>잔고 및 보유종목 현황</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>731</width>
       <height>65</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <row/>
     <column>
      <property name="text">
       <string>예수금 (D+2)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총매입</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총평가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총수익률 (%)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>추정자산</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="ButtonBalanceReq">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>396</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>보유종목 현황 조회</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>-140</y>
       <width>611</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>잔고확인</string>
     </property>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>591</width>
        <height>61</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>예수금</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총매입</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총평가</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총손익</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총수익률</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>추정자산</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>현금주문</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="CheckBoxBalanceUpdate">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>400</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>실시간 조회</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>731</width>
       <height>301</height>
      </rect>
     </property>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>종목명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>종목코드</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>보유량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>매입가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>현재가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>평가손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>수익률 (%)</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>470</y>
      <width>1371</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>선정 종목 리스트</string>
    </property>
    <widget class="QTableWidget" name="RequestTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>1351</width>
       <height>361</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>40</number>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>10</y>
      <width>120</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1227</x>
      <y>13</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>계좌</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonOrder">
    <property name="geometry">
     <rect>
      <x>1295</x>
      <y>860</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>전체주문</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>39</y>
      <width>611</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>주문 현황</string>
    </property>
    <widget class="QTableWidget" name="OrderTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>591</width>
       <height>371</height>
      </rect>
     </property>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QPushButton" name="ButtonCheckOrder">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>396</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>주문현황 조회</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1388</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
